src/System.Text.Json/src/System/Text/Json/Serialization/JsonCamelCaseNamePolicy.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json.Serialization
+{
+    internal sealed class JsonCamelCaseNamePolicy : JsonPropertyNamingPolicy
+    {
+        protected override string ConvertName(string name)
+        {
+            if (string.IsNullOrEmpty(name) || !char.IsUpper(name[0]))
+            {
+                return name;
+            }
+
+            char[] chars = name.ToCharArray();
```

Comment:
Would it be possible to use the `ReadOnlySpan` cast on `string` in combination with `string.Create` to avoid the intermediate `char[]`?

Reply:
I guess for netstandard you'd have to used a fixed string instead...

Reply:
Not sure if it is worth the split code base for netstandard - this isn't really a hot path since this is cached during warm-up.

Reply:
Don't worry about it.

Reply:
Why is it not important here? We should still try to avoid unnecessary allocations on start up, no?

Reply:
I don't think adding an ifdef for a cached item here is worth it here.